### ISSUE TITLE
Can not open store in World of Tanks game

### ISSUE THREAD

**Issue by pnqx** [sandboxie-plus/Sandboxie#1001]
**Describe the bug**
I can not open store in World of Tanks game in sandboxie. 
When i click the store button, the store is loading indefinitely.

**To Reproduce**
1. Go to https://redir.wggames.cn/hd5d3cvf/ , download and install the game
2. Login game
3. Click  'store' (it names 军械军 in chinese)
4. store is loading indefinitely
![95W0xJq3Qw](https://user-images.githubusercontent.com/12554018/125442540-fb5840ad-6144-498a-a458-303165264add.png)

**Expected behavior**
see store items, like this screenshot:
![8YNDMux6tB](https://user-images.githubusercontent.com/12554018/125442493-97b4794f-720e-4c78-82fa-41f2e7d117a6.png)

**System details and installed software (please provide the following information):**
 - What is your Windows edition and version? 
 Windows 10 Pro 2004
 - What is your current Sandboxie edition and version? 
   Sandboxie Plus 0.8.2
 - Please mention in which version this bug got introduced, because it acts as a guide for testers and developers. 
   I only run the game in Sandboxie Plus 0.8.2
 - Please mention any security software running in the background, even if the real-time protection is disabled. 
   Microsoft Defender

**Additional context**
when i run `C:\Games\World_of_Tanks_CN\win64\cef_browser_process.exe --resource-path="C:/Games/World_of_Tanks_CN/res/cef/" --max-frame-rate=60  --locale="zh_cn" --id=13516` in sandboxie, the store can loaded.   the id 131516 is the process ID (PID) of game.

**Sandboxie configuration**

<details>
 
<summary>My initial sandboxie.ini settings</summary>
 
```

[GlobalSettings]

Template=RpcPortBindings
Template=LastPass
Template=OfficeClickToRun
Template=7zipShellEx
Template=WindowsRasMan
Template=WindowsLive
Template=OfficeLicensing
ActivationPrompt=n
FileRootPath=X:\Sandbox\%USER%\%SANDBOX%
SeparateUserFolders=y
KeyRootPath=\REGISTRY\USER\Sandbox_%USER%_%SANDBOX%
IpcRootPath=\Sandbox\%USER%\%SANDBOX%\Session_%SESSION%
EditAdminOnly=n
ForceDisableAdminOnly=n
ForgetPassword=n

[UserSettings_00F20079]

SbieCtrl_UserName=x
SbieCtrl_ShowWelcome=n
SbieCtrl_NextUpdateCheck=1612358073
SbieCtrl_UpdateCheckNotify=n
SbieCtrl_WindowCoords=2858,529,791,738
SbieCtrl_ActiveView=40021
SbieCtrl_TerminateWarn=n
SbieCtrl_ShortcutNotify=n
SbieCtrl_AutoApplySettings=y
SbieCtrl_SettingChangeNotify=n
SbieCtrl_ExplorerWarn=n
SbieCtrl_HideWindowNotify=n
SbieCtrl_EnableLogonStart=y
SbieCtrl_EnableAutoStart=y
SbieCtrl_AddDesktopIcon=n
SbieCtrl_AddQuickLaunchIcon=n
SbieCtrl_AddContextMenu=y
SbieCtrl_AddSendToMenu=y
SbieCtrl_ExplorerNotify=n
SbieCtrl_ProcessViewColumnWidths=250,70,300
SbieCtrl_ShouldDeleteNotify=n
SbieCtrl_ReloadConfNotify=n
SbieCtrl_ProcSettingsNotify=y
SbieCtrl_RecoverTarget=C:\Users\x\Desktop
SbieCtrl_SaveRecoverTargets=y
SbieCtrl_EditConfNotify=n
SbieCtrl_TerminateNotify=n
SbieCtrl_AutoStartAgent=SandMan.exe


[WorldOfTank]

Enabled=y
ConfigLevel=9
AutoRecover=n
BlockNetworkFiles=y
RecoverFolder=%Desktop%
RecoverFolder=%Personal%
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
BorderColor=#00FFFF,ttl
Template=OpenBluetooth
Template=AutoRecoverIgnore
Template=Firefox_Phishing_DirectAccess
Template=Chrome_Phishing_DirectAccess
Template=LingerPrograms
Template=BlockPorts
Template=qWave
Template=FileCopy
Template=SkipHook

```

</details>

**Comment by pnqx** [comment]
> Could you try the latest [Release v0.8.8 / 5.50.8](https://github.com/sandboxie-plus/Sandboxie/releases/tag/0.8.8) and report if it works for your issue?

 I had tried Release v0.8.8 / 5.50.8 and it is not works for this issue.

**Comment by pnqx** [comment]
The issue has been solved on v1.0.5.

**Comment by pnqx** [comment]
If someone can't launch 360 Wargaming Game Center (wgc.exe)，you can add
```
OpenIpcPath=wgc.exe,\Sessions\1\BaseNamedObjects\{14126A2E-9B9F-43B0-A19D-05D846D638CF}
```
to sandbox setting.
